Automattic/jetpack
# Payments Block: Fix placeholder alignment and spacing issues

Fixes https://github.com/Automattic/view-design/issues/226

#### Changes proposed in this Pull Request:

* Fix spacing, alignment and font size issues with the placeholder of the payments block using Twenty Twenty One, and Twenty Twenty.

| Before        | After           | 
| ------------- |:-------------:| 
| <img width="624" alt="Screen Shot 2021-03-30 at 1 32 18 PM" src="https://user-images.githubusercontent.com/1464705/113053265-f21df380-915c-11eb-9569-1bf6e0734ecf.png"> |  <img width="626" alt="Screen Shot 2021-03-30 at 12 36 59 PM" src="https://user-images.githubusercontent.com/1464705/113053211-e29eaa80-915c-11eb-87be-7c79c77f5ae1.png"> |

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate Twenty Twenty One theme
* Insert payments block and click the button to add a new payment plan.
* Do the same with Twenty Twenty.
* Confirm the placeholder design matches the above.
